src/test/tests/cveApplicabilityTree.test.ts

```diff
@@ -0,0 +1,182 @@
+import * as vscode from 'vscode';
+import * as path from 'path';
+import { assert } from 'chai';
+import { SourceCodeCveTreeNode, SourceCodeCveTreeNodeDetails } from '../../main/treeDataProviders/sourceCodeTree/sourceCodeCveNode';
+import { SourceCodeFileTreeNode } from '../../main/treeDataProviders/sourceCodeTree/sourceCodeFileTreeNode';
+import { Severity } from '../../main/types/severity';
+import { SourceCodeRootTreeNode } from '../../main/treeDataProviders/sourceCodeTree/sourceCodeRootTreeNode';
+import { ConnectionManager } from '../../main/connect/connectionManager';
+import { LogManager } from '../../main/log/logManager';
+import { ScanCacheManager } from '../../main/scanCache/scanCacheManager';
+import { ScanLogicManager } from '../../main/scanLogic/scanLogicManager';
+import { TreesManager } from '../../main/treeDataProviders/treesManager';
+import { createScanCacheManager } from './utils/utils.test';
+import { SourceCodeTreeDataProvider } from '../../main/treeDataProviders/sourceCodeTree/sourceCodeTreeDataProvider';
+import { IIssueCacheObject } from '../../main/types/issueCacheObject';
+import { DependenciesTreeNode } from '../../main/treeDataProviders/dependenciesTree/dependenciesTreeNode';
+import { GeneralInfo } from '../../main/types/generalInfo';
+import { IIssueKey } from '../../main/types/issueKey';
+import { IssuesDataProvider, VulnerabilityNode } from '../../main/treeDataProviders/issuesDataProvider';
+import { DependencyDetailsProvider } from '../../main/treeDataProviders/dependencyDetailsProvider';
+import { TreeDataHolder } from '../../main/treeDataProviders/utils/treeDataHolder';
+import { PackageType } from '../../main/types/projectType';
+import { IProjectDetailsCacheObject } from '../../main/types/IProjectDetailsCacheObject';
+
+describe('Cve Applicability Tree Tests', () => {
+    let workspaceFolders: vscode.WorkspaceFolder[];
+    let tmpDir: vscode.Uri = vscode.Uri.file(path.join(__dirname, '..', 'resources', 'cveApplicability', 'project'));
+    let dependenciesTreeNode: DependenciesTreeNode;
+    let logManager: LogManager = new LogManager().activate();
+    let dummyScanCacheManager: ScanCacheManager = createScanCacheManager();
+
+    let treesManager: TreesManager = new TreesManager(
+        [],
+        new ConnectionManager(logManager),
+        dummyScanCacheManager,
+        {} as ScanLogicManager,
+        logManager
+    );
+    before(async () => {
+        let generalInfo: GeneralInfo = new GeneralInfo('odin', '1.2.3', [], tmpDir.path, 'asgard');
+        dependenciesTreeNode = new DependenciesTreeNode(generalInfo);
+        workspaceFolders = [
+            {
+                uri: tmpDir,
+                name: '',
+                index: 0
+            } as vscode.WorkspaceFolder
+        ];
+    });
+
+    it('Source Code Cve Tree Node', () => {
```

Comment:
All "it" should be lowercase. Let's make this change in all other unit tests in this file.
```suggestion
    it('Source code CVE tree node', () => {
```